tests/installation/agama_reboot.pm

```diff
@@ -38,9 +42,63 @@ sub get_agama_install_console_tty {
     return 7;
 }
 
+sub verify_agama_auto_install_done_cmdline {
+    # for some remote workers, there is no vnc access to the install console,
+    # so we need to make sure the installation has completed from command line.
+
+    # live root can support ssh access
+    my $timeout = 200;
+    while ($timeout > 0) {
+        if (script_run("journalctl -u agama | grep 'Install phase done'") == 0) {
+            record_info("agama install phase done");
+            return;
+        }
+        sleep 20;
+        $timeout = $timeout - 20;
+    }
+    die "Install phase is not done, please check agama logs";
+}
+
+sub agama_system_prepare_ipmi {
+    # Mount the root disk after installation, and do some prepare tasks:
+    # Configure serial console, enable root ssh access, etc
+
+    my $device = get_var("AGAMA_ROOT_TARGET_DEVICE", "/dev/sda");
+    # Use partition prefix for nvme devices
+    my $prefix = $device =~ /nvme/ ? "p" : "";
+    my $root_partition_id = 2;
+    my $root_partition = "${device}" . $prefix . $root_partition_id;
+    record_info("Device information", "Device: ${device}\nRoot partition: ${root_partition}");
+    assert_script_run("mount ${root_partition} /mnt");
+    # Set correct serial console to be able to see login in first boot
+    my $sol_console = is_aarch64 ? get_var('IPXE_CONSOLE', 'ttyAMA0') : get_var('IPXE_CONSOLE', 'ttyS1');
```

Comment:
`IPXE_CONSOLE` is in format `IPXE_CONSOLE=ttyAMA0,115200`.  It means it will not work.
Also logic with default values if confusing, Use `get_required_var`, every bare metal worker should have it, and split it to get correct tty.